packages/code-editor/src/components/CodeEditor/CodeEditor.stories.tsx

```diff
@@ -11,8 +17,6 @@ import {
   PopUp,
   Fullscreen,
 } from "@hitachivantara/uikit-react-icons";
-import { useState } from "react";
-import { Modal } from "@mui/material";
```

Comment:
We need prettier/ESLint plugin to settle on an import order 😄

Reply:
Yes, I agree. I'll open a PR with this.